introduction.md

```diff
@@ -0,0 +1 @@
+Hi! I'm Alex Pawlowski, and I am learning how to better use Github. Thanks for the free weekend!
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.